content/docs/forms.md

```diff
@@ -0,0 +1,214 @@
+
+Typically, when using local forms, you would call the `useLocalForm` hook inside the component that renders a specific page (this would be the page component in Gatsby or Next.js). In this case, the form would be added to the sidebar when the user is on that page, and removed once they navigate away.
+
+### _useGlobalForm_
+
+`useGlobalForm` will create a form and add it to the _global menu_ region of the sidebar. This area of the sidebar is intended for forms that persist across all pages on a site.
+
+Note that calling `useGlobalForm` in a non-global context will still cause the form to be added or removed like a local form.
+
+### _useForm_
+
+`useForm` will create a form, but will not add it to the sidebar UI at all. You might use this for content that will be [edited inline](/docs/inline-editing), if you don't want the form to also appear in the sidebar.
+
+## Calling Form Hooks
+
+The signature of a form hook looks something like this:
+
+```javascript
+const [modifiedValues, form] = useForm(formConfig, watchedVars)
+```
+
+All three basic form hooks follow this same structure.
+
+### Hook Return Values
+
+Like other React hooks, the form hooks enclose their return data in an array, expecting developers to assign these values via destructuring.
+
+The first piece of data returned (`modifiedValues` in the above example) is an object containing all the data that is made editable by the form. As users edit data in the form, the values in this object change.
+
+The second piece of data (`form` in the above example) is an object representing the form.
+
+<!-- TODO expand upon form object and demonstrate a use case where you might need it -->
+
+### Form Configuration
+
+The first argument that a form hook receives (`formConfig` in the above example) is an object used to configure the form. Forms in Tina are built upon the [Final Form](https://final-form.org/) library, and inherit all of Final Form's configuration options.
+
+You can see the all of Final Form's form config options in the [Form Config Documentation](https://final-form.org/docs/final-form/types/Config), but the following options will most commonly be used when creating a form:
+
+| key             | description                                         |
+| --------------- | --------------------------------------------------- |
+| `initialValues` | An object containing the initial state of the form. |
+| `onSubmit`      | A function that runs when the form is saved.        |
+
+In addition to Final Form's options, Tina's form hooks accept the following additional configuration options:
+
+```typescript
+interface FormOptions<S> {
+  id: any
+  label: string
+  fields: Field[]
+  loadInitialValues?: () => Promise<S>
+  reset?(): void
+  actions?: any[]
+  meta?: {
+    [key: string]: string
+  }
```

Comment:
I don't think this is worth mentioning.

```suggestion
```

Reply:
In fact, I don't know why this exists. This should be removed from `Form`

Reply:
I think this was originally how we were gonna do global forms at one point. I'll remove any reference to meta